src/grimoirelab/core/runner/commands/run.py

```diff
@@ -91,24 +98,89 @@ def server(ctx: Context, devel: bool):
 
 
 @run.command()
+@click.argument('task-types', nargs=-1)
 @click.option('--workers',
               default=10,
               show_default=True,
               help="Number of workers to run in the pool.")
-def eventizers(workers: int):
-    """Start a pool of eventizer workers.
+def worker_pool(task_types: str, workers: int):
```

Comment:
I think we should keep `eventizers` command and create a new one (maybe `archivists`) to store the data (in this case on OpenSearch). I know your proposal it's more generic but the other one gives us more flexibility in terms of configuration and usability because users will know what to run.